.env.template

```diff
@@ -0,0 +1,2 @@
+JURNAL_COMPANY_API_KEY=
+JURNAL_API_URL=https://sandbox-api.jurnal.id
```

Comment:
Why do we use environment variable for a gem configuration? I think it's better to use [configuration pattern](https://brandonhilkert.com/blog/ruby-gem-configuration-patterns/), so the user will have flexibility to configure. It's up to the user whether they want to use environment variable or some other technique.